### PR TITLE
6.8.2 changelog

### PR DESCRIPTION
**Pull request by arbll**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+-------
+
+Release on: 2018-12-19
+
+.. _Release Notes_6.8.2_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Fix a null pointer exception that would happen when the agent stopped tailing a container
```
Comment (olivielpeau) [inline CHANGELOG.rst]: as a customer, do I know what `tailing a container` means? :)

**Review by olivielpeau — COMMENTED** [review]
left a question

File: CHANGELOG.rst (hunk 1 above)
Comment (achntrl) [reply CHANGELOG.rst]: We might say `stopped reading logs from a container`

Comment (arbll) [reply CHANGELOG.rst]: I think we should also avoid using `null pointer exception` which does not tell us if the agent crashes or not

Comment (achntrl) [reply CHANGELOG.rst]: ```Fix a bug where the agent would crash when the agent stopped reading logs from a container``` ?

Comment (arbll) [reply CHANGELOG.rst]: Let's replace the second `agent` by `it`. 

`stopped reading logs from a container`: I would be confused by what this means if I did not have the context. Is this happening only when a container exits ?

**Pull request merged**